app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,134 @@
+@import url("https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500&family=Montserrat:wght@400;700&display=swap");
+
+body {
+  font-family: "Montserrat", sans-serif;
+  background-color: #f5f6f8;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+.main-text {
+  font-family: "Dancing Script", cursive;
```

Comment:
## OPTIONAL 
* This is optional because the previous CR missed these requirements but i will strongly suggest that you implement these as they are part of the requirement for your capstone project 
* The colors and font family used are different from the given design 
![image](https://github.com/IsmailChangezi/snap-scan/assets/52497246/0906a5df-0e7d-4ece-89aa-82792e8bcd2f)
![image](https://github.com/IsmailChangezi/snap-scan/assets/52497246/90d59355-ec96-45ef-bbc2-b27534a0a67d)